pkgs/development/python-modules/aioresponses/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, python
+, pbr
+, aiohttp
+, ddt
+, asynctest
+, pytest_3
+}:
+
+buildPythonPackage rec {
+  pname = "aioresponses";
+  version = "0.6.0";
+  disabled = !(pythonAtLeast "3.5");
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ii1jiwb8qa2y8cqa1zqn7mjax9l8bpf16k4clv616mxw1l0bvs6";
+  };
+
+  buildInputs = [ pbr aiohttp ddt asynctest pytest_3 ];
```

Comment:
You're sure this doesn't work with the normal `pytest`?

Reply:
Yes. For a start, the package setup.py tests_requires say 3.1.3, which is a big hint. And if I "jailbreak" that dependency and use pytest 4 then all the tests fail.